dotCMS/src/main/webapp/html/portlet/ext/contentlet/publishing/view_publish_tool.jsp

```diff
@@ -311,3 +330,30 @@
 
     </form>
 </div>
+
+<div dojoType="dijit.Dialog" id="deleteBundleActions" title='<%= UtilMethods.escapeSingleQuotes(LanguageUtil.get(pageContext, "bundle.delete.title" )) %>'>
+    <table class="sTypeTable">
+        <tr>
+            <td id="selectedBundlesBtnContainer">
+                <button id="deleteSelectedBundles" dojoType="dijit.form.Button" class="dijitButton" onClick="deleteSelectedAudits()">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)